src/js/components/settings/organization/organization.js

```diff
@@ -134,21 +136,23 @@ export const Organization = ({
           }
         />
       </List>
-      <div className="flexbox center-aligned">
-        <FormControlLabel
-          className={`margin-bottom-small ${classes.ssoToggle}`}
-          control={<Checkbox checked={!isResettingSSO && (hasSingleSignOn || isConfiguringSSO)} onChange={onSSOClick} />}
-          label="Enable SAML single sign-on"
-        />
-        {isResettingSSO && !isConfiguringSSO && (
-          <>
-            <Button onClick={onCancelSSOSettings}>Cancel</Button>
-            <Button onClick={onSaveSSOSettings} disabled={!hasSingleSignOn} variant="contained">
-              Save
-            </Button>
-          </>
-        )}
-      </div>
+      {isEnterprise && (
```

Comment:
One request from @alfrunes, would it be possible to hide it unless version is "next" similarly to what we have with the IoT Core integration?